lib/vyasa_web/components/contexts/read.ex

```diff
@@ -198,18 +199,15 @@ defmodule VyasaWeb.Context.Read do
    When we actually create the sheaf that contains the marks, they may be published as public or private,
    regardless of whether the intent is a) or b) above.
 
-  [TBD: @ks0m1c]
-  TO VERIFY These are the invariants that will hold true for a single user within a single session:
-  1) at any point in time, there's only one OR zero non-draft sheaf that is active
-  2) at any point in time, there's only one draft sheaf that is active
-
-
   [TBD: @ks0m1c]
 
   There are two main ways this reply to context can be set (implicitly):
   1) because it has been set in another mode (discuss mode) ==> the db is used to mediate this.
-      => init_reply_to_context() reads from DB
-  2) via url params, as permalinked ==> this will take precendence over the db-based determining of reply_to_context
+      => init_reply_to_context() reads from DB.
+      By looking at the currently active draft sheaf, check it's parent id:
+      1A: the parent is nil: so no reply_to sheaf
+      1B: non-nil then load that parent sheaf as reply_to
+  2) TODO  @ks0m1c via url params, as permalinked ==> this will take precendence over the db-based determining of reply_to_context
```

Comment:
if its a binding to sheaf across both modes discuss and read

Reply:
we shouldnt override -> allow user for conscious reply_to actions

tldr allow for binding but no inference of user intent towards reply to

kiv, pop stash of current drafting table context